### ISSUE TITLE
--json option prints more than JSON

### ISSUE THREAD

**Issue by marcoskirsch** [ofek/pypinfo#40]
I was hoping to use the output of the script to spit out JSON I can bring in later. I invoke this way:

```
pypinfo --json --order download_date --start-date -365 --days 365 --limit 365 foo     date > ~/foo.json
```

File `/foo.json` doesn't contain valid JSON because there's some stuff before it:

```
Served from cache: False
Data processed: 151.63 GiB
Data billed: 151.63 GiB
Estimated cost: $0.75

[
  {
    "download_count": 24,
    "download_date": "2018-02-04"
  },
  {
    "download_count": 49,
    "download_date": "2018-02-03"
  },
  {
...
```

Workaround is to strip everything before the first `[`. Something like:
```
json = read_data[read_data.find('['):]
```

**Comment by hugovk** [comment]
That'll be from https://github.com/ofek/pypinfo/commit/7a21b81b9d1e225ce39c59361537c5e1c6c2d3d3, added in 9.0.0, and shouldn't be hard to fix.

In the meantime, another workaround is to install version 8.0.0.

**Comment by ofek** [comment]
This was an oversight on my part. I'll fix this tonight!

**Comment by ofek** [comment]
Fixed in `11.0.0`
